### ISSUE TITLE
Old action nodes behave different then new ones

### ISSUE THREAD

**Issue by canedje** [zachowj/node-red-contrib-home-assistant-websocket#1642]
### Describe the bug

If I change an old action node and want to add a TTS speaker it is available at the selection list.
![image](https://github.com/user-attachments/assets/8ec2c2e1-3474-4147-b6dd-cfa3e4382eef)
If I create a new action node the Unifi speakers are not available in the selection list. (TTS for Unifi is not working in both situation)
![image](https://github.com/user-attachments/assets/796edc61-2e29-41f4-9f1f-fdc4c2a0c48e)

### To Reproduce

Just create a new action node

### Expected behavior

The same behavior between both and the same action nodes

### Screenshots

see above

### Environment Information
Version: 0.74.0

Home Assistant version: 2024.10.1
Companion version: 4.1.1

Node-RED version: 4.0.4
Docker: yes
Add-on: no

Node.js version: v20.18.0 x64 linux
OS: Linux 6.1.103-Unraid x64

**Comment by zachowj** [comment]
The top screenshows the action as "media_player.play_media" and the bottom screenshot shows "media_player.media_play"

**Comment by canedje** [comment]
Sharp!!!
I didn’t see that. But both are working?

**Comment by rlyons20** [comment]
seems to be something funny...
<img width="1003" alt="Screenshot 2024-10-10 at 9 28 47 AM" src="https://github.com/user-attachments/assets/0054fadc-0e4b-46cd-a988-435c4af46170">

**Comment by zachowj** [comment]
@rlyons20 that subflow has been updated to work with the action node. <https://zachowj.github.io/node-red-contrib-home-assistant-websocket/cookbook/actionable-notifications-subflow-for-android.html>

**Comment by zachowj** [comment]
> I didn’t see that. But both are working?

Yes, they both show entities. Do entities show if you don't filter with the search box?

**Comment by canedje** [comment]
No. The media_play is not showing the entity.
See second picture above

**Comment by zachowj** [comment]
> No. The media_play is not showing the entity.

I was asking if you don't have the search box filled with a filter does it show any entities?

**Comment by canedje** [comment]
Sorry, did misunderstood.
Yes it is giving a list of mediaplayers, except the Unifi players. They are not showing up

**Comment by zachowj** [comment]
In Home Assistant, under **Developer Tools > States**, you can search for the entity that isn't showing up. Once you find it, look for the `supported_features` attribute and share the number. This will help determine what capabilities the entity has and might clarify why it isn't appearing where expected.

![image](https://github.com/user-attachments/assets/61b3093e-141e-492a-acdf-98dc17e12af2)

**Comment by zachowj** [comment]
TL;DR: Those entities don't support the `media_play` action.

Here are the target filters for `media_player.media_play`:
[Link to the filter code](https://github.com/home-assistant/core/blob/8ba14ef113a072650c8abce26da9290872eb8192/homeassistant/components/media_player/services.yaml#L76-L81)

And here's the definition for `PLAY`:
[Link to the constant](https://github.com/home-assistant/core/blob/8ba14ef113a072650c8abce26da9290872eb8192/homeassistant/components/media_player/const.py#L214C12-L214C17)

If you do a bitwise AND between the `supported_features` (136708) and `PLAY` (16384), you'll see that the entity does not support the `PLAY` feature (16384).

![image](https://github.com/user-attachments/assets/78fb06e9-ba7b-4ddb-84a6-8a0bb80a67f7)

**Comment by zachowj** [comment]
I don't have any Unifi devices but shouldn't you be using the "tts" domain?

**Comment by canedje** [comment]
I tried that also without result.
The TTS domain worked before

**Comment by zachowj** [comment]
TTS should work fine the only change there was that it requires the `entity_id` to be in the data field and not the target field.

**Comment by canedje** [comment]
Yes. But it doesn't
